agents/postgres/pgstatmonitor/pgstatmonitor_models.go

```diff
@@ -0,0 +1,410 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pgstatmonitor
+
+/*
+pgDefault
+	Rows              int64          `reform:"rows"`
```

Comment:
Does pg monitor 0.6 had only this column?

Reply:
Thanks removed this comment to not confuse people. It was done for me to be able to refactor.